documentatie/functionaliteit/versie-1.0-gr.md

```diff
@@ -35,51 +36,45 @@
 - De uitslag van het GSB wordt gegenereerd als proces-verbaal in PDF-formaat.
 - De uitslag kan na de eerste zitting worden gecorrigeerd met behulp van een corrigendum.
 
-### Invoeren van uitslagen CSB
-
-- De uitslag van het GSB kan worden geïmporteerd als XML-bestand volgens de EML_NL-standaard.
-- Uitslagen worden ingevoerd volgens het vierogenprincipe. Dit betekent dat ze twee keer worden ingevoerd door twee verschillende invoerders. Een import van een EML_NL-bestand telt ook als invoer, waardoor nog maar één handmatige invoer nodig is.
-- Verschillen tussen de twee invoeren oplossen: de coördinator verwijdert een van beide invoeren en laat deze herinvoeren.
-
-### Uitslagbepaling CSB
-
-- De zetelverdeling voor de gemeenteraad wordt uitgerekend en de zetels worden aan kandidaten toegewezen.
-- De uitslagen kunnen worden geëxporteerd als XML-bestand volgens de EML_NL-standaard.
-- De uitslag van het CSB wordt gegenereerd als proces-verbaal in PDF-formaat.
-
 ### Ondersteunende functies
 
 - Er is ondersteuning voor het gebruik van Abacus op meerdere werkstations.
 - De Abacus-server kan de afwezigheid van een internetverbinding detecteren (airgapdetectie).
 - Abacus is beschikbaar voor één open source operating system (Linux) en Microsoft Windows.
+- Er vindt logging van gebruikershandelingen plaats.
 
 ## Versie 1.0: zeer gewenst (should have)
 
 *Deze punten zijn zeer gewenst en worden geïmplementeerd als het kan, maar zonder is de software ook bruikbaar.*
 
-- De Abacus-clients kunnen de afwezigheid van een internetverbinding detecteren (airgapdetectie).
-- Er vindt logging van gebruikershandelingen plaats.
-- Bijhouden van statistieken over gebruik.
-- De coördinator GSB kan invoerders van het GSB beheren,
-  en de coördinator CSB kan invoerders van het CSB beheren.
-
+- Er is een installatieprogramma voor Microsoft Windows.
+- De coördinator GSB kan invoerders van het GSB beheren.
```

Comment:
Is dit niet een must have? / Hebben we dit niet impliciet al zolang we de rollenscheiding GSB/CSB nog niet gebouwd hebben?

Reply:
Ah ja zo, check, tx!